### ISSUE TITLE
Cant launch the game

### ISSUE THREAD

**Issue by epxcq** [bluemagic123/tModLoader#2517]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

So my friend and I were gonna try to play tmod loader and when he tried to open it after the update it didn't work, the first black screen would come up 
<img width="586" alt="bug" src="https://user-images.githubusercontent.com/93854004/172492907-a0c5fbc5-3593-4333-8293-f6ace1205425.png">
after that it would go to something else which is too fast to see and just close and would not open. We tried deleting the game and redownloading it we tried it on the website we tried a lot of things but nothing worked and nothing made it work. 

### Log File
I don't have his logs

### Steps to reproduce

I havent seen anyone else with this glitch and I don't know how

### Expected Behavior

the game should be opening

### Actual Behavior

the game is closing itself 

### Reproduction frequency

Unlikely to reproduce this bug

**Comment by G1org1owo** [comment]
Bumping this as a friend had the same problem just now and I spent over an hour looking around for solutions.

After reading the output a few times, we checked the dotnet folder and discovered he was missing everything but the dotnet exe. Both reinstalling the game and verifying file integrity did nothing and the only actual "fix" we came up with was me sharing my dotnet folder with him through drive.

I'm pretty sure this doesn't happen on every machine, otherwise it would have probably been resolved already, but for some reason sometimes the needed files are just not downloaded. The actual reason why is beyond me.
